studio/src/rte/rte-link-editor.js

```diff
@@ -133,7 +133,7 @@ export class RteLinkEditor extends LitElement {
     get #analyticsIdField() {
         const options = this.#isCheckoutLink
             ? Object.keys(CHECKOUT_CTA_TEXTS)
-            : ANALYTICS_LINK_IDS;
+            : [...ANALYTICS_LINK_IDS];
```

Comment:
this this is needed? isnt' `ANALYTICS_LINK_IDS` already an array?

Reply:
yeah i need to copy it, can't use the reference